xAPISGProcessor-SIMVA.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# xAPI-SG Processor SIMVA"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Processor of traces following the Experience API for Serious Games Profile (xAPI-SG)\n",
+    "\n",
+    "**xAPI-SG main reference:**\n",
+    "*Applying standards to systematize learning analytics in serious games.\n",
+    "Ángel Serrano-Laguna, Iván Martínez-Ortiz, Jason Haag, Damon Regan, Andy Johnson, Baltasar Fernández-Manjón\n",
+    "Computer Standards & Interfaces 50 (2017) 116–123, http://dx.doi.org/10.1016/j.csi.2016.09.014*\n",
+    "\n",
+    "Further info on GitHub wiki page: https://github.com/e-ucm/rage-analytics/wiki/xAPI-SG-Profile"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following code:\n",
+    "- expects as input a JSON file with a list of xAPI-SG statements, obtained from SIMVA\n",
+    "- analyzes the xAPI-SG statements and fills an adaptation of the default set of visualizations\n",
+    "https://github.com/e-ucm/rage-analytics/wiki/Default-visualizations-teacher"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SIMVA connection parameters \n",
+    "\n",
+    "### Defined in *ipyauth-keycloak-demo.env* file:\n",
```

Comment:
The default file uploaded to the repository contains different parameters. Which are the correct ones? Do these have to be added? Are the ones in the current uploaded file valid or do they have to be replaced?

Here is mentioned:
- base_url
- simva_ca_file
- client_id
- redirect_uri

The current uploaded file contain:
- base_url
- realm
- response_type
- client_id
- redirect_uri